10_files_lab/fileslab.txt

```diff
@@ -7,31 +7,72 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - human.h, monster.h
   - .highscore
 
+
 Write here the commands you used:
 
+nadavg@mews2659:~/lab$ cd
+nadavg@mews2659:~/lab$ mkdir lab
+nadavg@mews2659:~/lab$ cd lab
+nadavg@mews2659:~/lab$ >a.out
+nadavg@mews2659:~/lab$ >hero.c
+nadavg@mews2659:~/lab$ >enemy.c
+nadavg@mews2659:~/lab$ >game.c
+nadavg@mews2659:~/lab$ >main.c
+nadavg@mews2659:~/lab$ >human.h
+nadavg@mews2659:~/lab$ >monster.h
+nadavg@mews2659:~/lab$ > .highscore
+
+
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
-Write here the commands you used:
+Write here the commands you used: 
+
+nadavg@mews2659:~$ cd lab                                                                                                                                                                                                                                                      
+nadavg@mews2659:~/lab$ mkdir Music                                                                                                                                                                                                                                             
+nadavg@mews2659:~/lab$ mkdir Misc                                                                                                                                                                                                                                              
+nadavg@mews2659:~/lab$ mkdir Drivers
```

Comment:
Good. Also note mkdir can be used with multiple arguments such as: `mkdir Music Misc Drivers`